ui/eureka/eureka.js

```diff
@@ -18,10 +18,30 @@ let Options = {
       'Eureka Hydatos': '丰水之地',
     },
     ko: {
-      'Eureka Anemos': '아네모스 지대',
-      'Eureka Pagos': '파고스 지대',
-      'Eureka Pyros': '피로스 지대',
-      'Eureka Hydatos': '히다토스 지대',
+      'Eureka Anemos': '아네모스편',
+      'Eureka Pagos': '파고스편',
+      'Eureka Pyros': '피로스편',
+      'Eureka Hydatos': '히다토스편',
+    },
+  },
+  timeString: {
+    'for ([0-9]{1,3})m': {
+      'ko': '$1분 동안',
+    },
+    // 'for \?\?\?': {
+    //   'ko': '??? 동안',
+    // },
+    'in ([0-9]{1,3})m': {
+      'ko': '$1분 후',
+    },
+    // 'in \?\?\?': {
+    //   'ko': '??? 후',
+    // },
+    '([0-9]{1,3})m': {
+      'ko': '$1분',
+    },
+    '(;[0-9]{1,3})m': {
```

Comment:
I've added `;` in front, because it makes clear that there's no another string between weatherIcon and minute number.